python/composio/tools/toolset.py

```diff
@@ -1193,9 +1215,9 @@ def get_expected_params_for_user(
 
         raise ComposioSDKError(
             message=(
-                f"No existing integration found for `{str(app)}`, "
-                "Please create an integration and use the ID to "
-                "fetch the expected params."
+                f"No existing integration found for `{str(app)}`, with auth "
+                f"scheme {auth_scheme} Please create an integration and use the"
+                " ID to fetch the expected params."
```

Comment:
The error message construction in the `get_expected_params_for_user` method could be improved for better readability. Consider reformatting it as follows:

```python
raise ComposioSDKError(
    message=(
        f"No existing integration found for `{str(app)}` with auth scheme '{auth_scheme}'. "
        "Please create an integration and use the ID to fetch the expected params."
    )
)
```

This format provides clearer information about the error and maintains better line breaks.